### ISSUE TITLE
Compile releases on older OS?

### ISSUE THREAD

**Issue by burner1024** [WeiDUorg/weidu#215]
`./weidu: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by ./weidu)`
v249 doesn't run on Ubuntu 18.04, which has over a year to EOL.
Compiling is not a big issue, but I think it'd better to lower OS requirements still.

**Comment by FredrikLindgren** [comment]
Yeah, I've been meaning to, but it's work so I guess I've been putting it off until someone complained.

**Comment by FredrikLindgren** [comment]
Current CentOS Stream has GNU libc v2.34, which is older than my semi-current Fedora desktop (GNU libc v2.37. Is that old enough or should I keep looking?

**Comment by burner1024** [comment]
Well, U18.04 is EOL now, and 20.04 has libc 2.31, should be available in github actions still.
My case is 22.04 now, which has 2.35.

**Comment by FredrikLindgren** [comment]
CentOS Stream 8 is the oldest I can find, with v2.28. Debian oldstable is also on v2.31.

**Comment by burner1024** [comment]
Anything 2.35 and below is fine for me, can't speak for other people =).
